README.md

```diff
@@ -28,8 +28,8 @@ Two sample CSV files, for nodes and edges, are available [here](https://github.c
 
 > Make sure to have write access to the folder of Linkurious Enterprise as well as a Linkurious admin account.
 
-1. Download the `csv-importer.lke` plugin available [here](https://github.com/Linkurious/lke-plugin-csv-importer/releases/tag/v1.0.0).
-2. Copy `csv-importer.lke` file to the following path `<linkurious>/data/plugins`
+1. Download the `lke-plugin-csv-importer-1.0.1.lke` plugin available [here](https://github.com/Linkurious/lke-plugin-csv-importer/releases/tag/v1.0.1).
```

Comment:
it is better to replace the "-1.0.1" with something like "-{version number}" so it will be more generic and we don't have to change the readme each time we release a new version.
And for the link we can use "https://github.com/Linkurious/lke-plugin-csv-importer/releases" that redirect you to the last release.